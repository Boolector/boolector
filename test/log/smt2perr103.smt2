(declare-fun a () (Array (_ BitVec 16) (_ BitVec 1)))
(assert (store a a (_ bv0 32)))
