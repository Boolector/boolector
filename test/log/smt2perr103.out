log/smt2perr103.smt2:2:18: second argument of 'store' is an array
